webrtc-stats.html

```diff
@@ -612,8 +612,54 @@ <h3>
           the first RTCP packet is sent or received that refers to the <a>SSRC</a> of the <a>RTP stream</a>.
         </p>
         <p>
-          RTP monitored objects are not [= deleted =].
+          RTP monitored objects are deleted when the RTP stream is reconfigured
+          such that the {{RTCRtpStreamStats/ssrc}} or
+          {{RTCRtpStreamStats/codecId}} changes or if the
+          {{RTCRtpTransceiver}}'s {{RTCRtpTransceiver/currentDirection}} becomes
+          <code>"stopped"</code>. The monitored object is <i>not</i> deleted if
```

Comment:
nit: I find that the use of "this RTP stream" in the descriptions of RTCRtpStreamStats/codecId and RTCRtpStreamStats/transportId could be made clearer since "this" (the RTCRtpStreamStats object) and the associated RTP stream now have different lifetimes.

Otherwise this change seems sensible and LGTM.

Reply:
Updated the language to talk about RTP senders and RTP receivers being the things getting reconfigured. Removed the part about codecId changing.